### DISCUSSION TITLE
How to establish a connection to ADLS? How the vector store can be consumed by another app?

### DISCUSSION THREAD

**Discussion by jasneetkaurdeloitte** [SEMOSS/community#40] [Q&A] [answered]
Requesting the steps for the below:
How to establish a connection to ADLS?
How the vector store can be consumed by another app?

**Comment by saahithisk** [comment, marked as answer]
A vector store can be utilized by an application using the code snippets provided in the Usage tab of the Vector Catalog (CreateEmbeddingsFromDocuments, ListDocumentsInVectorDatabase, etc.)

**Reply by jasneetkaurdeloitte** [reply]
Thanks

**Reply by jasneetkaurdeloitte** [reply]
How to establish a connection to ADLS?
